dart/lib/src/client.dart

```diff
@@ -77,6 +74,15 @@ abstract class SentryClient {
     return _options.transport.send(event);
   }
 
+  SentryEvent _prepareEvent(SentryEvent event) => event.copyWith(
```

Comment:
can we do the `sdk` field here too?

like https://github.com/getsentry/sentry-java/blob/113af8211c07ef622a4e265cc6b3438a6194d9fe/sentry/src/main/java/io/sentry/MainEventProcessor.java#L86-L88

right now its done on SentryEvent ctor, but we ideally move to here as well:
https://github.com/getsentry/sentry-dart/blob/feature/unified-api/dart/lib/src/protocol/sentry_event.dart#L37

this means, options.sdk should already have this value set

Reply:
✅